docs/index.md

```diff
@@ -142,6 +148,32 @@ The URL at `<<iss value from Signed JWT>>` SHALL NOT include a trailing `/`. For
 }
 ```
 
+### Certificates
+
+X.509 certificates can be used by issuers to indicate the issuer's participation in a PKI-based trust framework.
+
+If the Verifier supports PKI-based trust frameworks and the Health Card issuer includes the `"x5c"` parameter in matching JWK entries from the `.keys[]` array,
+the Verifier establishes that the issuer is trusted as follows:
+1. Verifier validates the leaf certificate's binding to the Health Card issuer by:
+    * matching the `<<iss value from Signed JWT>>` to the value
+of a `uniformResourceIdentifier` entry in the certificate's Subject Alternative Name extension
+(see [RFC5280](https://tools.ietf.org/html/rfc5280#section-4.2.1.6)), and
+    * verifying the signature in the Health Card using the public key in the certificate.
+2. Verifier constructs a valid certificate path of unexpired and unrevoked certificates to one of its trusted anchors
+ (see [RFC5280](https://tools.ietf.org/html/rfc5280#section-6)).
+
+
+### Key Management
+
+Issuers SHOULD generate new signing keys at least annually. 
+
+When an issuer generates a new key to sign Health Cards, the public key SHALL be added to the
+issuer's JWK set in its jwks.json file. Retired private keys that are no longer used to sign Health Cards SHALL be destroyed.
```

Comment:
```suggestion
issuer's JWK set in its jwks.json file. Retired private keys that are no longer used to sign Health Cards SHALL be destroyed.
```

Luis, this is problematic. A primary use-case is signing a JWS that's represented as a QR code on paper. Once we remove the public key from the jwks.json, we're invalidating all QR codes signed with that keypair. "no longer used to sign" isn't a sufficient timeframe to ensure that QR codes on paper are no longer in use.

Reply:
I think you may be over-interpreting @isaacvetter  -- the statement in the PR say that *private keys* are destroyed when retired; the public keys remain in the JWKS after key rotation. We can add language to clarify if this is ambiguous.

Reply:
Specifically the PR says:

> Older public key entries that are needed to validate previously signed health cards SHALL remain in the JWK set for as long as the corresponding health cards are clinically relevant. However, if a private signing key is compromised, then the corresponding public key SHALL be removed from the JWK set immediately.

... which I think is clear; when you've had a chance to review, @isaacvetter, see if you agree.